src/meta/processors/job/JobManager.cpp

```diff
@@ -671,7 +701,121 @@ ErrorOr<nebula::cpp2::ErrorCode, uint32_t> JobManager::recoverJob(
       return retCode.first;
     }
     for (size_t i = 0; i < jobKeys.size(); i++) {
-      jobKVs.emplace_back(std::make_pair(jobKeys[i], jobVals[i]));
+      totalJobKVs.emplace_back(std::make_pair(jobKeys[i], jobVals[i]));
+    }
+
+    // For DATA_BALANCE and ZONE_BALANCE job, jobs with STOPPED, FAILED, QUEUE status
+    // The following situations can be recovered:
+    // Queue: The job has not been executed, the machine restarted.
+    // FAILED: The failed jobid will be executed. FAILED and QUEUE jobs will not exist at the same
+    // time. STOPPED: If only one stopped jobId is specified, there will not be finished job or
+    // failed jobId after the job. If multiple jobIds are specified, only last jobId will be
+    // recovered. there will not be finished job or failed jobId after the job. The form in which
```

Comment:
What does `there will not be finished job or failed jobId after the job.`  mean?
You mean once a job is STOPPED, we can't add any job?

BTW `there will not be finished job or failed jobId after the job.` is duplicated in the comment

Reply:
`You mean once a job is STOPPED, we can't add any job?`  can add this type job
but the specified type job failed， can't add this type job.

The following example, in the order of the starttime of the job
`job1(stopped) job2(failed)`
recover job job1     failed
recover job job2     success

`job1(stopped) job2(fininshed) job3(stopped)`
recover job job1     failed
recover job job3     success
recover job job1, job3   Only job3 can recover

Reply:
👌 , the comment in code is not exactly same with the comment here, which confuses me ...

Reply:
yeah, my fault, I have updated the comment